src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -7,6 +7,9 @@ import Text from '../../Text';
 import Space from '../../Space';
 import Flex from '../index';
 
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```

Comment:
why do you need this?

Reply:
https://github.com/razorpay/blade/pull/81#discussion_r402307800

Reply:
To suppress the error message that comes in test case where we pass more than on child and try to throw error.

Reply:
Will move it to that test case instead of beforeAll

Reply:
Yes please move it to that particular test case